.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+node_modules
```

Comment:
- [ ] As stated in the project requirement Kindly Make sure that the last file(test.md file) is actually ignored by git and is not present in your GitHub repository.

![Capture](https://user-images.githubusercontent.com/93598189/218445288-37eedaba-acfe-469f-b693-7618ff5658f2.PNG)